docs/cloud/bestpractices/multitenancy.md

```diff
@@ -0,0 +1,366 @@
+---
+slug: /cloud/bestpractices/multi-tenancy
+sidebar_label: 'Implement multi tenancy'
+title: 'Multi tenancy'
+description: 'Best practices to implement multi tenancy'
+---
+
+On a SaaS data analytics platform, it is common for multiple tenants, such as organizations, customers, or business units, to share the same database infrastructure while maintaining logical separation of their data. This allows different users to securely access their own data within the same platform
+
+Depending on the requirements, they are different ways to implement multi-tenancy. Below is a guide on how to implement them with ClickHouse Cloud.
+
+## Shared table  {#shared-table}
+
+In this approach, data from all tenants is stored in a single shared table, with a field (or set of fields) used to identify each tenant’s data. To maximize performance, this field should be included in the [primary key](/sql-reference/statements/create/table#primary-key). To ensure that users can only access data belonging to their respective tenants we use [role-based access control](/operations/access-rights), implemented through [row policies](/operations/access-rights#row-policy-management).
+
+> **We recommend this approach as this is the simplest to manage, particularly when all tenants share the same data schema.**
```

Comment:
and data volumes are moderate < 1TB

Reply:
not a per-requisite but if its small volumes this is the best way **if possible**

Reply:
added a note that this is recommended approach - especially for moderate data volumes